src/os/windows/mod.rs

```diff
@@ -365,25 +361,30 @@ impl<T> ::std::ops::Deref for Symbol<T> {
     fn deref(&self) -> &T {
         unsafe {
             // Additional reference level for a dereference on `deref` return value.
-            &*(&self.pointer as *const *mut _ as *const T)
+            &*(&self.pointer.unwrap() as *const _ as *const T)
         }
     }
 }
 
 impl<T> fmt::Debug for Symbol<T> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.write_str(&format!("Symbol@{:p}", self.pointer))
+        match self.pointer {
+            None => f.write_str("Symbol@0x0"),
+            Some(ptr) => f.write_str(&format!("Symbol@{:p}", ptr as *const ())),
+        }
     }
 }
 
-struct ErrorModeGuard(DWORD);
+struct ErrorModeGuard(u32);
 
 impl ErrorModeGuard {
     #[allow(clippy::if_same_then_else)]
     fn new() -> Option<ErrorModeGuard> {
+        use self::DebugApi::SEM_FAILCRITICALERRORS as SEM_FAILCE;
```

Comment:
I’d say import this where the `SEM_FAILCE` was defined originally, and maybe drop the rename?

Reply:
Done